tests/qsfs_check_metrics/qsfs_test.go

```diff
@@ -46,23 +44,20 @@ func TestMultiNodeDeployment(t *testing.T) {
 	ygg_ip := terraform.Output(t, terraformOptions, "ygg_ip")
 	assert.NotEmpty(t, ygg_ip)
 
-	status := false
-	status = tests.Wait(ygg_ip, "22")
-	if status == false {
-		t.Errorf("public ip not reachable")
-	}
+	err = tests.Wait(ygg_ip, "22")
+	assert.NoError(t, err)
 
 	verifyIPs := []string{ygg_ip, metrics}
```

Comment:
metrics is already on yggdrasil endpoint right? if so, no need to check on ygg_ip